src/controllers/OnStatusController.js

```diff
@@ -0,0 +1,25 @@
+import LoggingService from '../services/LoggingService';
+import StatusServices from '../services/StatusService';
+import genericResponse from '../utilities/GenericResponse';
+import authVerifier from '../utilities/SignVerify/AuthHeaderVerifier';
+import LookUpService from '../services/LookUpService';
+
+const onStatus = async (req, res) => {
+  const logger = LoggingService.getLogger('OnStatusController');
+  logger.debug(`on_status called with ${JSON.stringify(req.body)}`);
+  logger.debug(req.body.context.bpp_id);
+  const publicKey = await LookUpService.getPublicKeyWithUkId(req.body.context.bpp_id);
```

Comment:
This looks wrong. The function says getPublicKeyWithUkId but the parameter is bpp_id

Reply:
Resolved